extension/manifest.json

```diff
@@ -16,9 +16,11 @@
       "js": ["page-extractor.js", "arcs-content-script.js"]
     }
   ],
+  /*
```

Comment:
Not sure what to do here. I run a regular server at port 80, so these things hardwired to 8080 dont' work for me. Probably this should be put back as before for this PR, so it works for Berni's demo, but it's not a good solution. Matt and I both have reservations about NewTabPage (NTP) being Arcs, seems like it would be better to optionally launch arcs from the extension button?